scipy/spatial/ckdtree.pyx

```diff
@@ -1175,8 +1175,9 @@ cdef class cKDTree:
         ----------
         x : array_like, last dimension self.m
             An array of points to query.
-        k : integer
-            The number of nearest neighbors to return.
+        k : integer or list
+            The number of nearest neighbors to return,
+            or a list of ranks of nearest neighbor to return.
```

Comment:
Adding an example of what this does and/or some more detailed explanation of the benefits in the ``Notes`` section would be good.